### PR TITLE
Fix risk budget labels. 

### LINKED ISSUE #382: Risk budgets don't have parens at the end of the descriptions
image

### PR DESCRIPTION
**Pull request by blanchardjeremy**
Fixes #382 and #400

### CODE REVIEW

**Comment by blanchardjeremy** [comment]
Nevermind. Not ready yet. See #382 discussion thread.

**Comment by blanchardjeremy** [comment]
Now it's ready.

File: src/components/calculator/RadioControl.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  source: Array<CheckBoxFormValue>
+  value: string | number
+  label?: string
+  header?: string
+  helpText?: string
+  popover?: JSX.Element
+  className?: string
+}> = (props) => (
+  <Form.Group controlId={props.id}>
+    {(props.label || props.header) && (
```
Comment (beshaya) [inline src/components/calculator/RadioControl.tsx:18]: Hm, It'd be nice to refactor this label stuff that's shared between RadioControl and FormControl. Maybe in a follow-up.

**Review by beshaya — APPROVED** [review]

File: src/components/calculator/RadioControl.tsx (hunk 1 above)
Comment (blanchardjeremy) [reply src/components/calculator/RadioControl.tsx]: Done. :)

**Pull request merged**